### DISCUSSION TITLE
Simple ways to make cooking more fun

### DISCUSSION THREAD

**Discussion by DrEnzyme** [space-wizards/space-station-14#24631] [Ideaguys]
I'm looking for an intro project to get started coding in the game, and I thought "why not add some more recipes" with the goal of encouraging cross-department collaboration, giving the chef more recipes that don't just help with nutrition and giving cooking more things to do with the weird shit botany always makes. Some possible ideas:

* Gateau: cake with gatfruit. Is very funny.
* Lemon and Poppyseed cake: Delicious, also takes advantage of poppy's healing qualities.
* Eye Burger: Another one of the body piece burgers. Contains a tiny bit of oculine for those that have blinded themselves.
* Teppanyakoi: Something to do with all the Koibeans botany keeps making.

I feel like these (or recipes like them) help to give the cook a bit more actual purpose within the game, and help to bridge the disconnect that often happens when botany decides to do something crazy. I'd also like to see more recipes that require weird ingredients so that you have a reason to go out and talk to the other departments or look around the station. 

Anyways. Thoughts, suggestions? Or should I just get to it?

**Comment by deltanedas** [comment]
lemon cake already exists just needs a recipe if it doesnt have one

**Reply by DrEnzyme** [reply]
You're right, I was thinking more of poppies though. If it's too close I could make poppyseed bagels instead, I don't think bagels exist at all. Maybe a dough slice and a poppy in a microwave...

Also fun fact, you can test positive on a drug test if you eat certain varieties of poppy seed beforehand.

**Comment by DrEnzyme** [comment]
![bagel](https://github.com/space-wizards/space-station-14/assets/6075909/b25180fb-f23e-4d62-b044-344b860e90b1)
![bagel-poppy](https://github.com/space-wizards/space-station-14/assets/6075909/23ec6ee8-7ee3-4fe4-87a5-6734e89bf606)

Some concepts

**Comment by UbaserB** [comment]
Cooking is under a restriction as seen here #24346 so any changes you make may be reverted.

**Reply by DrEnzyme** [reply]
Dang. Thanks UbaserB. I might still use this as a coding test but stick to one food item then. That way I won't be so upset if they get deleted.
